client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -142,6 +144,20 @@ class EditorConfirmationSidebar extends React.Component {
 		);
 	}
 
+	renderPublishDateSelector() {
```

Comment:
The use of 'selector' confused me briefly here -- was thinking of a Redux selector. Suggest changing to 'control' for consistency with privacy control.